nightfall-optimist/src/utils/auth.mjs

```diff
@@ -0,0 +1,5 @@
+export default function auth(req, res, next) {
+  const token = req.headers['x-app-token'];
+  if (token === process.env.AUTH_TOKEN) return next();
+  return res.status(401).send('Unauthorized');
+}
```

Comment:
I discussed this with @israelboudoux , and in principle I also agree to avoid duplication. However, in this case there would be no reuse for it. As of now this is the only service using API auth so maybe we could leave that change if we ever implement the same for other services